apps/labs/components/Post/FeaturedPosts/FeaturedPosts.tsx

```diff
@@ -40,7 +40,10 @@ export const FeaturedPosts: FC<TFeaturedPostsProps> = ({ posts }) => {
                 </Link>
               </h4>
               <p className="font-sans text-[1.2rem] font-normal leading-[2.7rem]">
-                By {post.meta.author.nickName} {post.meta.published}
+                By {post.meta.authors[0].nickName}
+                {post.meta.authors.length > 1 &&
+                  `+ ${post.meta.authors.length - 1} more`}
+                {post.meta.published}
```

Comment:
Same here as below, looking forward to design, let's separate the date from the authors:

```suggestion
                {` — ${post.meta.published}`}
```

Reply:
@gabalafou - What thoughts here? (Is this still relevant?)

Reply:
Ah, it's not - the date is now on its own line.

Reply:
Yeah, I think we're fine here